### PR TITLE
fixed ukis content issues

### PR DESCRIPTION
**Pull request by yoganandkunche**
### What is the context of this PR?
Fixed UKIS content related issues

### How to review 
Please find attached PDF for issues in UKIS schema and re-test them.

[UKIS_content_Observations.pdf](https://github.com/ONSdigital/eq-survey-runner/files/714464/UKIS_content_Observations.pdf)

### CODE REVIEW

File: app/data/1_0001.json [hunk 1]
```diff
@@ -5130,25 +5130,19 @@
...
                     "sections": [
                         {
-                            "description": "<p>Submission</p><p>You will not be able to access or change your answers on submitting the questionnaire</p><p>If you wish to review your answers please select the relevant Completed sections</p><p>If you do not submit your questionnaire, your responses will be collected when the online questionnaire closes</p>",
+                            "description": "",
                             "id": "ready-to-submit-completed-section",
                             "questions": [
                                 {
                                     "answers": [],
-                                    "description": "",
+                                    "description": "<ul><li><p>You will not be able to access or change your answers on submitting the questionnaire</p></li><li><p>If you wish to review your answers please select the relevant Completed sections</p></li><li><p>If you do not submit your questionnaire, your responses will be collected when the online questionnaire closes</p></li></ul>",
```
Comment (samiwel) [inline app/data/1_0001.json]: I think this is okay for now. As a rule, I think we're trying not to introduce HTML tags into the schema. However, the `description` key does not support the `list` construct (in the same way as `guidance`, for example).

Comment (yoganandkunche) [reply app/data/1_0001.json]: Addressed your comment. please review

**Review by LJBabbage — CHANGES_REQUESTED** [review]
We need to be really carefully here, the JSON used in this PR is pretty old and has been moved around a bit, 9.9 is still about no innovation, which shows its age. We need to get the other 2 PR's in first before highlighting issues

**Comment by iwootten** [comment]
Closing this until #831 and #840 have been merged to master and any remaining issues can be identified.

**Pull request closed without merging**